rdr_service/offline/biobank_samples_pipeline.py

```diff
@@ -354,7 +358,45 @@ def _query_and_write_withdrawal_report(exporter, file_path, report_cover_range,
     exporter.run_export(file_path, withdrawal_report_query, backup=True)
     logging.info(f"Completed {file_path} report.")
 
-def _query_and_write_reports(exporter, now, report_type, path_received,
+
+def _build_query_params(start_date: datetime):
+    code_dao = CodeDao()
+    race_question_code = code_dao.get_code(PPI_SYSTEM, RACE_QUESTION_CODE)
+    native_american_race_code = code_dao.get_code(PPI_SYSTEM, RACE_AIAN_CODE)
+
+    return {
+        "race_question_code_id": race_question_code.codeId,
+        "native_american_race_code_id": native_american_race_code.codeId,
+        "biobank_id_prefix": get_biobank_id_prefix(),
+        "pmi_ops_system": _PMI_OPS_SYSTEM,
+        "ce_quest_system": _CE_QUEST_SYSTEM,
+        "kit_id_system": _KIT_ID_SYSTEM,
+        "tracking_number_system": _TRACKING_NUMBER_SYSTEM,
+        "n_days_ago": start_date,
+        "dv_order_filter": 0
+    }
+
+
+def _query_and_write_received_report(exporter, report_path, query_params, report_predicate):
+    received_report_select = _RECONCILIATION_REPORT_SELECTS_SQL
+    if config.getSettingJson('enable_biobank_manifest_received_flags', default=False):
```

Comment:
I don't see an entry for this on the prod config? 
Can we also make an entry for this on `rdr_service/config.py` as to conform to how we pull the other config data?
`if config.getSettingJson(config.ENABLE_BIOBANK_RECIEVED_FLAGS, default=False):`

Reply:
Sorry for the delayed response. There's no entry for it yet, since we're not sure yet when they would want it to run, but I'll be updating the config when we know.

And thanks for catching the config strings. Updated the code to read from the module.

Reply:
Nice! Thanks!